common/src/GtinHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Valued\WordPress;
+
+class GtinHandler {
+    const SUPPORTED_PLUGINS = [
+        'woocommerce-product-feeds/woocommerce-gpf.php' => null,
+        'customer-reviews-woocommerce/ivole.php' => '_cr_gtin',
+        'product-gtin-ean-upc-isbn-for-woocommerce/product-gtin-ean-upc-isbn-for-woocommerce.php' => '_wpm_gtin_code',
+        'woo-product-feed-pro/woocommerce-sea.php' => '_woosea_gtin',
+    ];
+    private $product;
+
+    public function __construct(\WC_Product $product = null) {
+        $this->product = $product;
+    }
+
+    public function hasActivePlugin(): bool {
+        foreach (self::SUPPORTED_PLUGINS as $plugin) {
+            if (is_plugin_active($plugin)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public function getGtin(): ?string {
+        if (is_plugin_active('woocommerce-product-feeds/woocommerce-gpf.php')) {
+            return $this->handleGpf();
+        }
+        return $this->getGtinFromMeta($this->getGtinMetaKey());
+    }
+
+    private function getGtinMetaKey(): string {
+        foreach (array_filter(self::SUPPORTED_PLUGINS, 'strlen') as $plugin => $key) {
+            if (is_plugin_active($plugin)) {
+                return $key;
+            }
+        }
+        return '_wwk_gtin_code';
```

Comment:
You've hardcoded this meta key, but it should depend on the plugin name.  `wwk` doesn't make sense for the TP plugin.

You can do `_{$plugin->getSlug()}_gtin` or `_{$plugin->getOptionName('gtin')}`

No real need to call it a "GTIN code" because that's a "Global Trade Item Number code".  "number code" is double, so it's just a "GTIN".